packages/generator/src/generators/page-generator.ts

```diff
@@ -41,12 +41,17 @@ export class PageGenerator extends Generator<PageGeneratorOptions> {
       modelNames: this.options.modelNames,
       ModelName: this.options.ModelName,
       ModelNames: this.options.ModelNames,
+      importModelNames: this.getPathWithContext(),
```

Comment:
Happy to change to a better name could not think about any other atm.

Reply:
What about `modelNamesPath`?

Reply:
Changed here 783027e